.github/pull_request_template.md

```diff
@@ -30,7 +30,51 @@ _Include relevant screenshots_
 
 #### After
 
-### Checklist:
+### Review your work
+Please self review your work before requesting further review. _Wait until you reach the next screen to check off the boxes._
+
+#### Preview
+Use your local or the preview build.
```

Comment:
Maybe we should reword it so they review both local and preview builds? 
The local would be self reviewed before the PR is opened, the preview would be after

Reply:
Ok I changed it to just use your local so that they are able to do things like check the terminal etc. I think the preview build can just be checked in the PR review guide version of this.